portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/StructuredContentFilteringAndSorting.testcase

```diff
@@ -39,7 +39,7 @@ definition {
 				ddmStructureId = "${ddmStructureId}",
 				label = "Text",
 				name = "Text",
-				priority = "1.0",
+				priority = "1.1",
```

Comment:
I changed the priority from 1.0 to 1.1 to try to resolve the priority conflict failures